swift/WorkflowUI/Sources/Container/ContainerView.swift

```diff
@@ -0,0 +1,203 @@
+#if canImport(SwiftUI) && canImport(Combine) && swift(>=5.1)
+
+import Workflow
+import ReactiveSwift
+import SwiftUI
+import Combine
+
+
+/// Hosts a Workflow-powered view hierarchy.
+///
+/// Example:
+///
+/// ```
+/// var body: some View {
+///     ContainerView(workflow: MyWorkflow(), onOutput: { self.handleOutput($0) }) { rendering in
+///         VStack {
+///
+///             Text("The value is \(rendering.value)")
+///
+///             Button(action: rendering.onIncrement) {
+///                 Text("+")
+///             }
+///
+///             Button(action: rendering.onDecrement) {
+///                 Text("-")
+///             }
+///
+///         }
+///     }
+/// }
+/// ```
+@available(iOS 13.0, *)
+public struct ContainerView<T: Workflow, Content: View>: View {
```

Comment:
This name seems pretty vague, can we call it `WorkflowContainerView` or something?

Reply:
`WorkflowView`? I know we used the term `container` internally, but I agree that it isn't a very useful term here.